### ISSUE TITLE
Update to latest esbuild?

### ISSUE THREAD

**Issue by satlom** [mhsdesign/esbuild-plugin-lightningcss-modules#3]
It is not possible to install this plugin with the latest esbuild 0.19.4 due to this packages dependencies on 0.16 and 0.17. Do you have any plans to fix this?

**Comment by mhsdesign** [comment]
yes i do but at this very moment i dont have the time for it. Eventually i will do it myself but feel free to open a pr and test if it works with version 18 and 19.

I remember reading the release notes but im not sure if there was anything breaking that might affect this plugin.
As declared here: https://github.com/evanw/esbuild/issues/2544 esbuild is currently moving quite fast and still breaking stuff from version to version, thats why i currently can only guarantee this plugin works with 16 and 17. Any other esbuild version would need to be tested first before i do an release ;)
